OpenIDConnectSwift/Info.plist

```diff
@@ -29,6 +18,17 @@
 	<string>1.0</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>com.okta</string>
```

Comment:
Agreed that a universal link would be ideal. Is this possible without having the developer select a Development Team?

If not, how about updating the url scheme to include the `clientID`:
```
com.okta.clientID
```